host-spec/.gitignore

```diff
@@ -0,0 +1,3 @@
+host-spec.tex
```

Comment:
Might want to remove this, otherwise LGTM.

Reply:
Also, maybe add newlines between each CI step (in the yaml file).

Reply:
I will see if the .tex files are still generated correctly and make a decision based on that. The yaml formatting can be changed, but if we do so, we should change it in all workflow files.